code/datums/traits/negative.dm

```diff
@@ -328,6 +328,23 @@
 		if(I.fingerprintslast == quirk_holder.ckey)
 			quirk_holder.put_in_hands(I)
 
+/datum/quirk/phobia
+	name = "Phobia"
+	desc = "You have a debilitating fear you just can't seem to overcome."
+	value = -2
+	gain_text = null
+	lose_text = null
+	medical_record_text = "Patient has a deep rooted phobia."
+
+/datum/quirk/phobia/add()
```

Comment:
is there a difference if you use `on_spawn()`?

Reply:
Id imagine admins wouldn’t be able to add the quirk to someone

Reply:
nah they call `on_spawn` with their add-minabuse.

To my understanding, on_spawn should be the "default"

Reply:
on spawn provides no noticeable difference

Reply:
dunno why they're different anyways but 🤷‍♂